delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java

```diff
@@ -84,6 +84,10 @@ public int getSizeInBytes() {
     if (sizeInBytes < 0) {
       // the size of all other fields are fixed, only return the dynamic size of row and previousRow
       sizeInBytes = operation.getSizeInBytes() + computeSizeInBytes(previousRow);
+      // delete event didn't count the size of row in the DMLOperation
+      if (operation.getType() == DMLOperation.Type.DELETE) {
```

Comment:
This will be used while emitting metrics too where we dont want to count row size for delete events.

Reply:
DMLOperation.getSizeInBytes is used for metrics :
https://github.com/data-integrations/delta/blob/develop/delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java#L48

not this one :)

Reply:
and this one also calculate the size of previousRow which metrics don't care.

Reply:
ah ok. thanks Sean for clarification!